### PR TITLE
Use phpstan v9 and fix reported errors.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

**Comment by antograssiot** [comment]
@ADmad you forgot to update or push Travis config I think

**Comment by ADmad** [comment]
@antograssiot Yeah i did forget to update travis config :P

File: src/Phinx/Db/Adapter/PostgresAdapter.php [hunk 1]
```diff
@@ -981,9 +981,6 @@ protected function getIndexSqlDefinition(Index $index, $tableName)
             $indexName = $index->getName();
         } else {
             $columnNames = $index->getColumns();
-            if (is_string($columnNames)) {
```
Comment (ADmad) [inline src/Phinx/Db/Adapter/PostgresAdapter.php]: Can someone confirm that removing this check is safe? 

Docblock of `\Phinx\Db\Table\Index::getColumns()` says it returns `array` and such a `is_string()` isn't present in `MysqlAdapter`, so I think removing this check here and in `SqlServerAdapter` shouldn't be a problem.

**Comment by ADmad** [comment]
[This](https://travis-ci.org/cakephp/phinx/jobs/324472303#L591) remaining failure is to due incorrect docblock in a symfony component. I had submitted a fix which was merged and should be available in their next bugfix release.

**Comment by lorenzo** [comment]
Do you mind fixing this @ADmad?

**Comment by ADmad** [comment]
@lorenzo Done.

**Pull request merged**